src/components/street-geo.js

```diff
@@ -7,17 +7,19 @@ const MAPBOX_ACCESS_TOKEN_VALUE =
 
 AFRAME.registerComponent('street-geo', {
   schema: {
+    maps: {
+      type: 'string',
+      default: 'google3d',
+      oneOf: ['google3d', 'mapbox2d', 'osm3d']
+    },
     longitude: { type: 'number', default: 0 },
     latitude: { type: 'number', default: 0 },
     elevation: { type: 'number', default: null }, // deprecated
     orthometricHeight: { type: 'number', default: null },
     geoidHeight: { type: 'number', default: null },
     ellipsoidalHeight: { type: 'number', default: null },
-    maps: {
-      type: 'string',
-      default: 'google3d',
-      oneOf: ['google3d', 'mapbox2d', 'osm3d']
-    }
+    clippingEnabled: { type: 'boolean', default: true },
```

Comment:
Probably should be false in the final version for backward compatibility with existing scenes.